docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
       - fam
 
   fam-flyway:
-    image: flyway/flyway:latest-alpine
+    image: flyway/flyway:latest
     container_name: fam-api-migrations
```

Comment:
the "flyway/flyway:latest-alpine" image doesn't work for Mac, so changed

Reply:
This is just a docker-compose change so won't impact anything other than local developer setup. I'm fine with this.